brontology/config.py

```diff
@@ -1,4 +1,25 @@
-# noinspection PyUnresolvedReferences
-from typing import Union, Optional
+import spacy
+from spacy import Language
 
-MODEL_TYPE = "en_core_web_trf"
+
+class _Model:
+    """Lazy loader for the spacy language model."""
+
+    type_ = "en_core_web_trf"
+
+    def __init__(self):
+        self._inst: Language | None = None
+
+    def load(self):
+        """Explicitly loads the model."""
+        self._inst = spacy.load(self.type_)
+
+    @property
+    def inst(self) -> Language:
```

Comment:
Das hier ist ein Lazy-Loader Pattern. Die werden dir vor allem bei ML sehr oft begegnen, weil damit langandauernde Prozesse herausgezögert werden können, bis sie aktiv gebraucht werden.

Die Idee ist, dass man in einem Basisattribut den wahren Wert des Objekts (In diesem Fall das Sprachmodell in `._inst`) speichert, man aber auf dieses Basisattribut nie direkt zugreift, sondern nur über das richtige Attribut (`.doc`).

Dieses richtige Attribut kümmert sich dann dadrum, bei seinem ersten Aufruf (Aber nicht vorher!) das Sprachmodell reinzuladen, und es dann zurückzugeben. Ab dem Zeitpunkt ist das Modell permanent in diesem Attribut gespeichert, und weitere Aufrufe erzeugen nicht erneut den Rechenaufwand.